src/IRsend.cpp

```diff
@@ -897,6 +901,14 @@ bool IRsend::send(const decode_type_t type, const uint64_t data,
       sendMidea24(data, nbits, min_repeat);
       break;
 #endif  // SEND_MIDEA24
+#if SEND_MILESTAG2
+      case MILESTAG2SHOT:
+        sendMilesShot(data, nbits, min_repeat);
+        break;
+      case MILESTAG2MSG:
+        sendMilesMsg(data, nbits, min_repeat);
+        break;
+#endif  // SEND_MILESTAG2
```

Comment:
Per other comments, lets just use a single common `sendMilestag2()` function rather that two separate ones as they are identical except for nr of bits.